homeassistant/components/deconz/light.py

```diff
@@ -275,6 +278,21 @@ def __init__(self, device, gateway):
             if light.ZHATYPE == Light.ZHATYPE:
                 self.update_features(light)
 
+        for exclusive_color_mode in [COLOR_MODE_ONOFF, COLOR_MODE_BRIGHTNESS]:
+            if (
+                exclusive_color_mode in self._attr_supported_color_modes
+                and len(self._attr_supported_color_modes) > 1
+            ):
+                self._attr_supported_color_modes.remove(exclusive_color_mode)
+
+    @property
+    def hs_color(self) -> tuple | None:
+        """Return the hs color value."""
+        try:
+            return super().hs_color
+        except TypeError:
+            return None
```

Comment:
Should we add a test to cover this?

Reply:
I'll do that when I redo the tests if thats Ok